### PR TITLE
Correct transaction costs in segwit example

### PR DESCRIPTION
**Pull request by murchandamus**
The transaction fees in the example were off.
2-of-3 P2SH inputs are between 293 and 297 bytes (293 bytes if both
signatures have low-R, 296 bytes if one signature has high-R, 297 bytes
if both signatures have high-R) and P2SH outputs are 32 bytes.

2-of-3 P2WSH (native segwit) inputs are between 104 and 104.5 vbytes
(416–418 WU, depending on low-R/high-R as above) and P2WSH outputs are
43 bytes.

The transaction header has 10 bytes for non-segwit transactions and 42
WU for segwit transactions.

Source:
https://github.com/BitGo/unspents/blob/master/src/dimensions.ts#L69

I have updated the numbers in the example and amended the conclusion
which did not match the new numbers. Calculation details follow.

-----

Let's assume that the wallet is using signature grinding and will always
produce signatures with low-R:

•Without Segregated Witness:
Transaction A using 2-of-3 P2SH inputs and P2SH outputs:
3\*293 + 10 + 2\*32 = 953
953 \* 30 = 28,590

Transaction B using 2-of-3 P2SH inputs and P2SH outputs:
2\*293 + 10 + 3\*32 = 692
692 \* 30 = 20,760

•With Segregated Witness:
Transaction A using 2-of-3 P2WSH (native segwit) inputs and outputs:
3\*104 + 10.5 + 2\*43 = 408.5
408.5 \* 30 = 12,255

Transaction B using 2-of-3 P2WSH (native segwit) inputs and P2WSH
outputs:
2\*104 + 10.5 + 3\*43 = 347.5
347.5 \* 30 = 10,425

### CODE REVIEW

**Comment by aantonop** [comment]
Great PR thanks!

Can someone else double-check the calculations? I'd want a second review before merging it. @wbinns could you take a look?

Also @Xekyo please fix a couple of capitalization errors in the English text in the PR. A grammar checker can help with that if English is not your primary language!

**Comment by murchandamus** [comment]
@aantonop: I noticed that you don't capitalize "set" in "UTXO set" and the incorrectly capitalized "The" after the comma. Please let me know if you saw another and didn't mean "couple" literally. :+1: 

I also realized that I had staged but not committed the amendment of the transaction sizes to assume signature grinding, so I corrected the sizes. The calculation in the commit message now matches the text.

**Review by wbnns — APPROVED** [review]
@Xekyo 

LGTM; thank you!

**Comment by murchandamus** [comment]
I updated the text because the discussion did not fit the correct numbers, but since the statement is now much weaker than before, I was wondering whether the text should be changed more dramatically. I'd be happy to help with numbers for a different example if the section got updated more comprehensively, or could take a stab at rewriting that block if bigger changes are welcome.

**Comment by aantonop** [comment]
Bigger changes are absolutely welcome. Making a more nuanced point more
clearly is better than striving to make a point that no longer makes sense.

**Pull request merged**
